src/Sylius/Bundle/ApiBundle/Command/SendOrderConfirmation.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+use Sylius\Component\Order\Model\OrderInterface;
+
+class SendOrderConfirmation
+{
+    /** @var OrderInterface */
+    protected $order;
+
+    public function __construct(OrderInterface $order)
```

Comment:
the command should contain simple data, IMO instead `OrderInterface` it should be order token value.

Reply:
Yes, this will fail badly when you have async command, it will die with fatal and problem about deserialization of that object.